package.json

```diff
@@ -23,6 +23,8 @@
   "author": "Gregor Martynus (https://twitter.com/gr2m)",
   "license": "MIT",
   "devDependencies": {
+    "@graphql-codegen/cli": "^1.19.2",
+    "@graphql-codegen/typescript": "^1.18.0",
```

Comment:
I thin we should also add a `"types"` key to the package.json as per https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
We've already had `types` field for `index.d.ts`, so I `export * from './scheme'` in `index.d.ts`